pkgdb2/api/extras.py

```diff
@@ -625,7 +625,8 @@ def api_koschei():
 
 
 @API.route('/dead/package/<pkg_name>/<clt_name>')
-def api_dead_package(pkg_name, clt_name):
+@API.route('/dead/package/<namespace>/<pkg_name>/<clt_name>')
```

Comment:
This won't be backward compatible, we need either to support the old route or to add a second one